packages/block-editor/src/components/iframe/style.scss

```diff
@@ -0,0 +1,17 @@
+.block-editor-iframe__container {
+	width: 100%;
+	height: 100%;
+	overflow-x: hidden;
+}
+
+.block-editor-iframe__scale-container {
+	height: 100%;
+}
+
+.block-editor-iframe__scale-container.is-zoomed-out {
+	$container-width: var(--wp-block-editor-iframe-zoom-out-container-width, 100vw);
+	$prev-container-width: var(--wp-block-editor-iframe-zoom-out-prev-container-width, 100vw);
+	width: $prev-container-width;
+	// This is to offset the movement of the iframe when we open sidebars
+	margin-left: calc(-1 * (#{$prev-container-width} - #{$container-width}) / 2);
+}
```

Comment:
@MaggieCabrera This file appears to have been deleted on `wp/6.7` branch. But your PR includes it. Do you know what's going on there?

Reply:
I think the problem is we are missing this PR https://github.com/WordPress/gutenberg/pull/65977

Reply:
Do we want to backport #65977 too?

Reply:
I don't know. Asked there. The dependencies here are getting really hard to follow what with work moving forward in Gutenberg Plugin.

Reply:
While #65977 isn't strictly necessary for 6.7, the risks from backporting are low and bugfixes like this are being made in the CSS that was moved. So I'd say we should backport it just to make this one easier.